resources/external_features/scripts/normalize_hic_ctcf_features.R

```diff
@@ -0,0 +1,22 @@
+## Normalize CTCF PET and Hi-C loop count features
```

Comment:
For organization, i think it would be better to put these scripts still in the `workflow/scripts` directory, so that we don't have scripts scattered across different directories. You can add a subdirectory there, called `external_features` to keep these contained